File: Day 5/Passing Data Property ObjC/Passing Data Property ObjC/ViewController.m
//
//  ViewController.m
//  Passing Data Property ObjC
//
//  Created by Consultant on 8/24/22.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

//@IBAction func onButtonTap() {
//        let vc = storyboard?.instantiateViewController(withIdentifier: "vc2") as! ViewController2
//        vc.passText = "Hello World"
//
//        present(vc, animated: true, completion: nil)
//    }

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

- (IBAction)onButtonTap:(id)sender {
    ViewController *vc = [self.storyboard instantiateViewControllerWithIdentifier:@"vc2"];
    UIStoryboard *storyboard = [UIStoryboard ViewController2:bundle:nil];
    GCHConnectViewController* gchCVC = [storyboard instantiateViewControllerWithIdentifier:@"vc2"];
}

@end

File: Day 5/Passing Data Property Segues Swift/Passing Data Property Segues Swift/ViewController.swift
//
//  ViewController.swift
//  Passing Data Property Segues Swift
//
//  Created by Consultant on 8/24/22.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }

    @IBAction func myButton(_ sender: Any) {
        self.performSegue(withIdentifier: "mySegue", sender: self)
    }
    
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
            let segueDestination = segue.destination as! ViewController2
            segueDestination.passTextSegue = "Hello World"
        }

    
}

